src/v2/ion/i18nTransformer.js

```diff
@@ -122,6 +122,35 @@ const I18N_OVERRIDE_MAPPINGS = {
   'oie.session.expired' : 'oie.idx.session.expired',
 };
 
+const getDisplayName = (remediation, defaultName) => {
+  return remediation.relatesTo?.value?.displayName || defaultName;
+};
```

Comment:
Thanks for doing this! This helper method will hopefully allow us to simplify a lot of our view complexity.

Since this isn't specific to `i18n` and retrieves an authenticator's `displayName`, I'd suggest moving this to `AuthenticatorUtil` instead.

Reply:
Since the `displayName` is a required attribute with remediation forms, we can probably get away with not adding a default value here.